config/cable.yml

```diff
@@ -1,11 +1,11 @@
 development:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: redis://redis:6379/1 # redis://localhost:6379/1
```

Comment:
why don't we use local

Reply:
redis refers to the redis service in docker-compose, for local development we need to update with localhost.
Maybe `<%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>` can be used for development url?